consensus/istanbul/backend/handler.go

```diff
@@ -266,10 +267,28 @@ func (sb *Backend) UpdateMetricsForParentOfBlock(child *types.Block) {
 	inParentSeal := childExtra.ParentAggregatedSeal.Bitmap.Bit(gpValSetIndex) != 0
 	if inParentSeal {
 		sb.blocksElectedAndSignedMeter.Mark(1)
+		sb.blocksElectedButNotSignedGauge.Update(0)
 	} else {
 		sb.blocksElectedButNotSignedMeter.Mark(1)
+		sb.blocksElectedButNotSignedGauge.Update(sb.blocksElectedButNotSignedGauge.Value() + 1)
 		sb.logger.Warn("Elected but didn't sign block", "number", number-1, "address", sb.ValidatorAddress())
 	}
+
+	// Report downtime events
+	if sb.blocksElectedButNotSignedGauge.Value() >= 12 {
```

Comment:
nit:  What do you think of replacing `12` with a named constant (perhaps that is already defined somewhere)?